src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] values = Color.values();
+        int index = new Random().nextInt(values.length);
```

Comment:
we do not need to create new Random object each time method is called, move it to class variable